web_accounts_list.json

```diff
@@ -4767,6 +4767,28 @@
            "category" : "music",
            "valid" : true
           },
+          {
+           "name" : "castingfr",
+           "check_uri" : "https://www.casting.fr/{account}",
+           "account_existence_code" : "200",
+           "account_existence_string" : ", artiste sur Casting.fr -",
+           "account_missing_string" : "Page introuvable",
+           "account_missing_code" : "404",
+           "known_accounts" : ["mamadou01", "serge"],
+           "category" : "misc",
+           "valid" : true
+          },
+          {
+           "name" : "vinted",
+           "check_uri" : "https://www.vinted.nl/member/general/search?search_text={account}"

```

Comment:
You need to add a trailing `,` to this row otherwise the entire JSON file is invalid.

Reply:
Added it for you. :)